src/components/Dialogs/Dialogs.jsx

```diff
@@ -2,45 +2,61 @@ import React from "react";
 import s from "./Dialogs.module.css";
 import { NavLink } from "react-router-dom";
 
-const DIALOGS = "/dialogs";
-
-const dialogItems = ["Руслана", "Роман", "Тарас", "Сірожа", "Андрій"];
-
 const DialogItem = (props) => {
+  let path = "/dialogs/" + props.id;
```

Comment:
```suggestion
  const path = "/dialogs/" + props.id;
```
і тут тоді зможеш ре ужити ту змінну діалокс

Reply:
зараз зроблю